controllers/articles.controller.js

```diff
@@ -0,0 +1,12 @@
+const { selectArticleById } = require('../models/articles.model')
+
+exports.getArticleById = (req, res, next) => {
+	const { article_id } = req.params
+	selectArticleById(article_id)
+		.then(({ rows }) => {
+			if (rows.length === 0)
+				return Promise.reject({ status: 404, msg: 'Not found' })
```

Comment:
This promise rejection functionality would be better placed in the model. We want our controllers to only receive the returned array if it is to be returned to the client on the response object. The model should reject the promise (or return the empty array to the controller if appropriate).